resources/fixtures/Clay_Paky/Clay-Paky-Alpha-Spot-575-HPE.qxf

```diff
@@ -151,28 +151,28 @@
  </Channel>
  <Channel Name="Gobo 2 rotation">
   <Group Byte="0">Gobo</Group>
-  <Capability Min="0" Max="127">Fixed position</Capability>
-  <Capability Min="128" Max="190">Fast to slow rotation</Capability>
-  <Capability Min="191" Max="192">Stop rotation</Capability>
-  <Capability Min="193" Max="255">Slow to fast rotation</Capability>
+  <Capability Min="0" Max="127" Preset="RotationIndexed">Fixed position</Capability>
+  <Capability Min="128" Max="190" Preset="RotationClockwiseFastToSlow">Fast to slow rotation</Capability>
+  <Capability Min="191" Max="192" Preset="RotationStop">Stop rotation</Capability>
+  <Capability Min="193" Max="255" Preset="RotationClockwiseSlowToFast">Slow to fast rotation</Capability>
```

Comment:
Maybe in this row of four presets: RotationIndexed/RotationClockwiseFastToSlow/RotationStop the fourth one needs to be RotationCounterClockwiseSlowtoFast? like in channel Gobo 1 Rotation. If so, some presets below need to be changed, too.

Reply:
I checked the manual and you are correct. @kluszon please change the preset on line **155** to RotationCounterClockwiseFastToSlow